code/modules/SCP/SCPs/SCP-895.dm

```diff
@@ -0,0 +1,47 @@
+#define MESSAGE_COOLDOWN 100
+
+/obj/structure/closet/coffin/scp895
+	name = "oak coffin"
+	var/redzone = 5
+	var/list/cooldown = list()
+	var/list/start = list()
+	var/list/users = list()
+
```

Comment:
You forgot certain defines like in SCP-106

/datum/scp/SCP_106
	name = "SCP-106"
	designation = "106"
	classification = KETER

Reply:
Fixed